### ISSUE TITLE
`cgi` is deprecated and will be removed in Python 3.13

### ISSUE THREAD

**Issue by merkys** [jamesrhester/pycifrw#12]
Python module `cgi` [is deprecated](https://peps.python.org/pep-0594/#deprecated-modules) and will be removed in Python 3.13. There is a single occurrence of it being used in pycifrw source, in `src/Programs/process-validate.py`:
https://github.com/jamesrhester/pycifrw/blob/344f88fd39cdeb8240df90e15131894e1452d2e3/src/Programs/process-validate.py#L11

It would be nice to have `cgi` replaced before Python 3.13 is released.

**Comment by jamesrhester** [comment]
I've added a comment to that file (a demonstration program) explaining that [legacy-cgi](https://pypi.org/p/legacy-cgi) should be installed. See commit 0eea3ae.

**Comment by merkys** [comment]
> I've added a comment to that file (a demonstration program) explaining that [legacy-cgi](https://pypi.org/p/legacy-cgi) should be installed. See commit [0eea3ae](https://github.com/jamesrhester/pycifrw/commit/0eea3aea5b914fcb2df5916aaccb287c0251b5a1).

Indeed, `legacy-cgi` is now used as drop-in replacement for `cgi` in Debian.
